setup.cfg

```diff
@@ -11,21 +11,18 @@ classifier =
     Development Status :: 1 - Planning
     License :: OSI Approved :: ISC License (ISCL)
     Operating System :: OS Independent
-    Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.4
-    Programming Language :: Python :: 3.5
-    Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
     Topic :: Software Development :: Libraries :: Python Modules
 
 [options]
 zip_safe = true
-python_requires = >=2.6,!=3.0,!=3.1,!=3.2,!=3.3
-setup_requires = setuptools>=36.2.2
+python_requires = !=2.7,!=3.0,!=3.1,!=3.2,!=3.3,!=3.4,!=3.5,!=3.6
```

Comment:
Would this be better as >=3.7?

Reply:
Yes >=3.7 would be better

Reply:
Hey! @uranusjr welcome back to the party :smiley: